btr-web/btr-main-app/interfaces/type-of-control.ts

```diff
@@ -0,0 +1,5 @@
+export interface ControlTypeI {
+    registeredOwner: boolean;
```

Comment:
rmv the semi colons pls. Also can you pls update the file name to follow the same naming convention as the other interfaces 'control-type-i.ts'